src/script.js

```diff
@@ -457,7 +445,7 @@ function calculateExpenses() {
     // Skrij opozorilo - napačna izbira odjemne skupine glede na izbran način obračuna
     let odjSk_et = document.getElementById('warn-odjSk-et');
     odjSk_et.style.display = 'none';
-    
+
     // Prikaži opozorilo - napačna izbira odjemne skupine glede na izbran način obračuna
     if (nacObr_selectVal == '1-tarif' && (odjSk_selectVal != "gospodinjstvo" && odjSk_selectVal != "bmm")) {
```

Comment:
always try to use "===" this checks type of value as well "==" deosn't check if it's a string, number etc.